core/src/test/java/org/bitcoinj/core/PeerTest.java

```diff
@@ -148,7 +148,7 @@ public void chainDownloadEnd2End() throws Exception {
         
         peer.startBlockChainDownload();
         GetBlocksMessage getblocks = (GetBlocksMessage)outbound(writeTarget);
-        assertEquals(blockStore.getChainHead().getHeader().getHash(), getblocks.getLocator().get(0));
+        assertEquals(blockStore.getChainHead().getHeader().getHash(), getblocks.getLocator().getHashes().get(0));
```

Comment:
Since `BlockLocator.getHashes().get(i)` is used here and there, maybe adding a `BlockLocator.getHash(i)` might be an idea?